yoticlient.go

```diff
@@ -100,22 +95,22 @@ func (client *Client) GetSdkID() string {
 // GetActivityDetails requests information about a Yoti user using the one time use token generated by the Yoti login process.
 // It returns the outcome of the request. If the request was successful it will include the users details, otherwise
 // it will specify a reason the request failed.
-func (client *Client) GetActivityDetails(token string) (ActivityDetails, []string) {
-	_, activity, errStrings := client.getActivityDetails(token)
-	return activity, errStrings
+func (client *Client) GetActivityDetails(token string) (ActivityDetails, error) {
+	_, activity, err := client.getActivityDetails(token)
+	return activity, err
 }
 
-func (client *Client) getActivityDetails(token string) (userProfile UserProfile, activity ActivityDetails, errStrings []string) {
+func (client *Client) getActivityDetails(token string) (userProfile UserProfile, activity ActivityDetails, err error) {
 
 	httpMethod := http.MethodGet
 	key, err := loadRsaKey(client.Key)
 	if err != nil {
-		errStrings = append(errStrings, fmt.Sprintf("Invalid Key: %s", err.Error()))
```

Comment:
errStrings was nil before this point, and we immediately return - so it can never be more than length 1